src/components/RecipeDetails/RecipeDetails.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import Breadcrumb from "../Breadcrumb/Breadcrumb";
+import { BsFillTagFill } from "react-icons/bs";
+import RecipeNutrition from "./RecipesNutrition";
+import RecipeIngredients from "./RecipeIngredients";
+
+const ProductDetails = ({ uri }) => {
+  const [recipe, setRecipe] = useState({});
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  const url = `https://api.edamam.com/api/recipes/v2/${uri}?app_id=77aa9220&app_key=0308e0fb57a38dfce56f6cdff2845a7a&type=public`;
```

Comment:
It's generally a best practice to put your credentials (such as API keys) in a `env` file. You can read more [here](https://create-react-app.dev/docs/adding-custom-environment-variables/)!